sorts/natural_sort.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+import re
+
+
+def natural_sort(input_list: list[str]) -> list[str]:
+    """
+    Sort the given list of strings in the way that humans expect.
+
+    The normal python sort algorithm sorts lexicographically,
```

Comment:
```suggestion
    The normal Python sort algorithm sorts lexicographically,
```
Python is a brand name, not a snake.